app/src/main/java/ch/epfl/cs311/wanderwave/ui/components/utils/BeaconPlacer.kt

```diff
@@ -0,0 +1,98 @@
+package ch.epfl.cs311.wanderwave.ui.components.utils
+
+import ch.epfl.cs311.wanderwave.model.data.Beacon
+import ch.epfl.cs311.wanderwave.model.data.Location
+import com.google.android.gms.maps.model.LatLng
+import kotlin.math.*
+import kotlin.random.Random
+
+private const val EARTH_RADIUS_M = 6371000.0
```

Comment:
Should we put this in the ressources ? I think it's fine as it's only related to this file, but we may have to do it, if we need it soon in another file

Reply:
True, if we use it somewhere else, we should add a file for the constants